<Type Name="TargetParameterCountException" FullName="System.Reflection.TargetParameterCountException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="a6447ebe9a33d0d3f3d8d75dca0d548b50ca7891" />
    <Meta Name="ms.sourcegitcommit" Value="d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="es-ES" />
    <Meta Name="ms.lasthandoff" Value="04/03/2018" />
  </Metadata>
  <TypeSignature Language="C#" Value="public sealed class TargetParameterCountException : ApplicationException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable sealed beforefieldinit TargetParameterCountException extends System.ApplicationException" />
  <TypeSignature Language="DocId" Value="T:System.Reflection.TargetParameterCountException" />
  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class TargetParameterCountException&#xA;Inherits ApplicationException" />
  <TypeSignature Language="C++ CLI" Value="public ref class TargetParameterCountException sealed : ApplicationException" />
  <AssemblyInfo>
    <AssemblyName>System.Reflection</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
    <AssemblyVersion>4.0.10.0</AssemblyVersion>
    <AssemblyVersion>4.1.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
    <BaseTypeName FrameworkAlternate="netframework-4.5">System.ApplicationException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary>
      <span data-ttu-id="94fa3-101">Excepción que se produce cuando el número de parámetros de una llamada no coincide con el número esperado.</span>
      <span class="sxs-lookup">
        <span data-stu-id="94fa3-101">The exception that is thrown when the number of parameters for an invocation does not match the number expected.</span>
      </span>
      <span data-ttu-id="94fa3-102">Esta clase no puede heredarse.</span>
      <span class="sxs-lookup">
        <span data-stu-id="94fa3-102">This class cannot be inherited.</span>
      </span>
    </summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="94fa3-103">`TargetParameterCountException` usa el COR_E_TARGETPARAMCOUNT HRESULT que tiene el valor 0x8002000E.</span><span class="sxs-lookup"><span data-stu-id="94fa3-103">`TargetParameterCountException` uses the HRESULT COR_E_TARGETPARAMCOUNT which has the value 0x8002000E.</span></span>  
  
 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Reflection</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="94fa3-104">Inicializa una nueva instancia de la clase <see cref="T:System.Reflection.TargetParameterCountException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-104">Initializes a new instance of the <see cref="T:System.Reflection.TargetParameterCountException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public TargetParameterCountException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Reflection.TargetParameterCountException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; TargetParameterCountException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Reflection</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="94fa3-105">Inicializa una nueva instancia de la clase <see cref="T:System.Reflection.TargetParameterCountException" /> con una cadena de mensaje vacía y el problema raíz de la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-105">Initializes a new instance of the <see cref="T:System.Reflection.TargetParameterCountException" /> class with an empty message string and the root cause of the exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="94fa3-106">`TargetParameterCountException` hereda de <xref:System.Exception>.</span><span class="sxs-lookup"><span data-stu-id="94fa3-106">`TargetParameterCountException` inherits from <xref:System.Exception>.</span></span> <span data-ttu-id="94fa3-107">Este constructor establece las propiedades del objeto Exception tal y como se muestra en la tabla siguiente.</span><span class="sxs-lookup"><span data-stu-id="94fa3-107">This constructor sets the properties of the Exception object as shown in the following table.</span></span>  
  
|<span data-ttu-id="94fa3-108">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="94fa3-108">Property</span></span>|<span data-ttu-id="94fa3-109">Valor</span><span class="sxs-lookup"><span data-stu-id="94fa3-109">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|`null`|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="94fa3-110">La cadena vacía ("").</span><span class="sxs-lookup"><span data-stu-id="94fa3-110">The empty string ("").</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public TargetParameterCountException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Reflection.TargetParameterCountException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; TargetParameterCountException(System::String ^ message);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Reflection</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="94fa3-111">Valor <see langword="String" /> que describe la razón por la cual se produjo esta excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-111">A <see langword="String" /> describing the reason this exception was thrown.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="94fa3-112">Inicializa una nueva instancia de la clase <see cref="T:System.Reflection.TargetParameterCountException" /> con su cadena de mensaje establecida en el mensaje dado y el problema raíz de la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-112">Initializes a new instance of the <see cref="T:System.Reflection.TargetParameterCountException" /> class with its message string set to the given message and the root cause exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="94fa3-113">`TargetParmeterCountException` hereda de <xref:System.Exception>.</span><span class="sxs-lookup"><span data-stu-id="94fa3-113">`TargetParmeterCountException` inherits from <xref:System.Exception>.</span></span> <span data-ttu-id="94fa3-114">Este constructor establece las propiedades de la `Exception` del objeto tal y como se muestra en la tabla siguiente.</span><span class="sxs-lookup"><span data-stu-id="94fa3-114">This constructor sets the properties of the `Exception` object as shown in the following table.</span></span>  
  
|<span data-ttu-id="94fa3-115">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="94fa3-115">Property</span></span>|<span data-ttu-id="94fa3-116">Valor</span><span class="sxs-lookup"><span data-stu-id="94fa3-116">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|`null`|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="94fa3-117">Cadena del mensaje.</span><span class="sxs-lookup"><span data-stu-id="94fa3-117">The message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public TargetParameterCountException (string message, Exception inner);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception inner) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Reflection.TargetParameterCountException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; TargetParameterCountException(System::String ^ message, Exception ^ inner);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Reflection</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="inner" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="94fa3-118">Mensaje de error que explica el motivo de la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-118">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <param name="inner">
          <span data-ttu-id="94fa3-119">Excepción que es la causa de la excepción actual.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-119">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="94fa3-120">Si el parámetro <c>inner</c> no es <see langword="null" />, la excepción actual se produce en un bloque <see langword="catch" /> que controla la excepción interna.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-120">If the <c>inner</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="94fa3-121">Inicializa una nueva instancia de la clase <see cref="T:System.Reflection.TargetParameterCountException" /> con el mensaje de error especificado y una referencia a la excepción interna que representa la causa de esta excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="94fa3-121">Initializes a new instance of the <see cref="T:System.Reflection.TargetParameterCountException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="94fa3-122">Una excepción que se produce como consecuencia directa de una excepción anterior debe incluir una referencia a esta última en la propiedad <xref:System.Exception.InnerException%2A>.</span><span class="sxs-lookup"><span data-stu-id="94fa3-122">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="94fa3-123">La propiedad <xref:System.Exception.InnerException%2A> devuelve el mismo valor que se haya pasado al constructor, o bien `null` en caso de que la propiedad <xref:System.Exception.InnerException%2A> no le haya proporcionado al constructor el valor de la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="94fa3-123">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="94fa3-124">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.Reflection.TargetParameterCountException>.</span><span class="sxs-lookup"><span data-stu-id="94fa3-124">The following table shows the initial property values for an instance of <xref:System.Reflection.TargetParameterCountException>.</span></span>  
  
|<span data-ttu-id="94fa3-125">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="94fa3-125">Property</span></span>|<span data-ttu-id="94fa3-126">Valor</span><span class="sxs-lookup"><span data-stu-id="94fa3-126">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="94fa3-127">Referencia a la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="94fa3-127">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="94fa3-128">Cadena con el mensaje de error.</span><span class="sxs-lookup"><span data-stu-id="94fa3-128">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
  </Members>
</Type>